package.json

```diff
@@ -48,50 +49,51 @@
           "singleQuote": true,
           "trailingComma": "es5",
           "bracketSpacing": false,
-          "jsxBracketSameLine": false
+          "jsxBracketSameLine": false,
+          "arrowParens": "always"
         }
       ]
     }
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-preset-shopify": "^16.1.0",
-    "eslint": "^4.7.1",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-shopify": "^16.2.0",
+    "eslint": "^4.15.0",
     "eslint-index": "^1.4.0",
     "eslint-plugin-shopify": "file:./.",
     "isparta": "^4.0.0",
     "istanbul": "^0.4.5",
-    "mocha": "^3.4.2",
-    "prettier": "^1.8.2",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "mocha": "^4.1.0",
+    "prettier": "^1.9.2",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0"
   },
   "peerDependencies": {
     "eslint": "<5 >=4.7.1"
   },
   "optionalDependencies": {
-    "prettier": "<2.0 >= 1.7.2"
+    "prettier": "<2.0 >= 1.9.2"
```

Comment:
I dislike that this installs `prettier` even when consuming projects don't use it.  I'd like to move it to `peerDependencies`.

Make sense, @ismail-syed and @lemonmade?

Reply:
Good point, but is this in the scope of this PR?

Reply:
I thought optional dependencies was the right thing — doesn't `peer` require the installation, even if unused?

Reply:
> I dislike that this installs prettier even when consuming projects don't use it

I agree. Optional dependencies aren't what we expected them to be. 

As per yarn [docs](https://yarnpkg.com/lang/en/docs/dependency-types/#toc-optionaldependencies)
> Optional dependencies are just that: optional. If they fail to install, Yarn will still say the install process was successful.

prettier will still be installed; in the chance that it fails, `yarn install` will still succeed. 

I suggest we change it to a peer dependency like we do with `eslint`.

Reply:
Here's a repo that confirms the optional/peer behaviour: https://github.com/GoodForOneFare/dependency-install-types

* Optionals are installed for all consumers
* Peers generate warnings during `yarn install`, and are not installed

Reply:
That sounds like something that should be discussed outside of this PR, unless I’m missing something?